code-challenges.test.js

```diff
@@ -37,10 +75,26 @@ var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+//create a test that makes the first letter of each word in the array upper case
+
+describe ('allCaps', ()=>{
+  it('makes first letter of each word in the array upper case', ()=>{
+    expect(allCaps(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(allCaps(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
+
+//Got RED
 
 
 // b) Create the function that makes the test pass.
+// I got stuck here. My thought process was to join the array so that I could access the specific indexes and use the toUpperCase function, and then turn
+// it into an array but I was unsuccessful.
 
+const allCaps = ()=>{
+  console.log(allCaps.join(""))
+  return
+}
```

Comment:
You've got the ground work laid out in your pseudo code.

allcaps needs to take in an argument of an array (the container for the words we are going to capitalize.) 

We write that like this 
```javascript
allCaps = (array) => {

}
```

- from here we can make some pseudo code and declare some variables 

```javascript
allCaps = (array) => {
// join the array 
let joinedArray = array.join(" ")
}
```
These are some of the steps we could take. I think the logic you want to follow though is capitalizing only the first letter of each word. In which case it would be good to think about the arrays and strings-- where we want them to be and how we want to access them (maybe using a higher order function).